### ISSUE TITLE
Feature Request: Portable keynavish

### ISSUE THREAD

**Issue by snowman** [lesderid/keynavish#13]
Thank you for the development of the Windows port of `keynav`,
I want to make this tool available on `scoop` repos.

I want to load `keynavrc` config on the same location where
`keynavish.exe` is located first, then other places, this makes it portable

And release compressed file including executable `keynavish.exe`
and `keynavrc` default config

*or* provide command-line arg `--install-config <filename>` to install config then exit, so without packaging with `keynavrc` default config, and release single executable like before

**Comment by lesderid** [comment]
Thanks for the suggestion, I'll try to find some time to look into it.

**Comment by lesderid** [comment]
Apologies for the delay.

Would it be sufficient if keynavish would just load `keynavrc` from the working directory first? Or is there some reason it should install the config somewhere as you suggested?

There already is a default config included (when no config exists), but it is different from the `keynavrc` file in the repository. (This matches the original keynav.)

**Comment by lesderid** [comment]
> Would it be sufficient if keynavish would just load `keynavrc` from the working directory first?

This was implemented in v1.5.0 (https://github.com/lesderid/keynavish/commit/ab028cf6d8bddb45a26d4f34615c870b32ec6e3a).

**Comment by snowman** [comment]
No, do not read from current working directory.

If you install portable applications into USB drive, you can carry the
USB drive around to run applications on different computers out of the box,
the file structures may looks like:

```
E:
└── apps
    ├── keynavish
    │   ├── keynavish.exe
    │   └── keynavrc
    ├── listary
    │   └── listary.exe
    ├── totalcommander
    │   └── totalcmd64.exe
    └── shortcuts
        ├── keynavish.lnk
        ├── listary.lnk
        └── totalcmd64.lnk
```

And by add path `E:\apps\shortcuts` to environment variable `PATH`,
you could type the name of shortcut in *anywhere* to run an app.

```
  set PATH=%PATH%;E:\apps\shortcuts
```

If `keynavish` reading config from current working directory,
you have to change the working directory where config stores before you run the app, 
otherwise the config would not be loaded, which is a bit inconvenient.

In Windows system, most portable program saves/reads the config
to/from the same directory where program is.

**Comment by lesderid** [comment]
> In Windows system, most portable program saves/reads the config
> to/from the same directory where program is.

This is now implemented in v1.6.0.
